python/tvm/relay/op/strategy/x86.py

```diff
@@ -364,9 +373,9 @@ def dense_strategy_cpu(attrs, inputs, out_type, target):
 def batch_matmul_strategy_cpu(attrs, inputs, out_type, target):
     """batch_matmul x86 strategy"""
     strategy = _op.OpStrategy()
-    if is_dynamic(out_type):
+    if is_dynamic(out_type) or is_auto_scheduler_enabled():
```

Comment:
I feel confused about the condition here. Does `is_dynamic(out_type)` just means the output shape has not been computed rather than this is a dynamic shape op?

Reply:
if `out_type` contains "Any", then it is a dynamic shape op